### DISCUSSION TITLE
postsrsd timeout ? what to do when postsrsd is down

### DISCUSSION THREAD

**Discussion by slater0013** [roehling/postsrsd#172] [General]
Hello,

When miltering message through postfix check_policy_service, It is possible to use a timeout !

Let's imagine postsrsd is down for any reason, new messages are refused (4xx code) because postfix cannot communicate with postsrsd.

Is there any way to make postsrsd NOT mandatory ? We need it for 1or 2% of the total messages we process. Would be bad to refuse them all in case the daemon is down :(

Thanks,
GS

**Comment by gilgongo** [comment]
Does `milter_default_action = accept` handle this (in `main.cf`)?
